hpat/tests/test_series.py

```diff
@@ -249,6 +257,26 @@ def test_impl(A):
         df = pd.DataFrame({'A': np.arange(n)})
         np.testing.assert_array_equal(hpat_func(df.A), test_impl(df.A))
 
+    def test_series_getattr_ndim(self):
```

Comment:
The same, the changes shouldn't be in this PR.

Reply:
I think it is master too.